.gitignore

```diff
@@ -4,7 +4,6 @@ _built
 TAGS
 *.DS_Store
 *~
-.vscode
```

Comment:
This would only conflict with project-specific settings, right? I don't have any setting specific to this project, so that's fine with me. All my VSC customizations are cross-project.

Reply:
That's correct -- it's only a problem if you already have folder-specific settings (or want to in the future for this project).

Reply:
I also use vscode and don't use a workspaces as of now either.  This works for me :+1:

Reply:
I use it, and I have saved some projects as workspaces (makes it easier to open with my keyboard maestro macros) but it looks like the settings in those workspaces are empty anyway.

Reply:
Sounds good. To clarify, VS Code uses the term "workspace" in a couple confusing ways.

If you have saved your folder to a "workspace" using the File menu options, then you have three levels of settings: User (global), Workspace (stored wherever the workspace itself is saved), and Folder (`.vscode` in the local directory).

If you just have a directory open (not a workspace), then you have two levels of settings: User (global), and Workspace (`.vscode` in the local directory).

What I'm proposing here (storing a `.vscode` dir in the repo itself) will only cause trouble with individual "Folder" settings (if you use a VS Code "Workspace"), or "Workspace" settings (if you just open a directory in VS Code). 😵